co2eq/meal.js

```diff
@@ -31,8 +32,10 @@ export const explanation = {
 
 export const modelCanRunVersion = 1;
 export function modelCanRun(activity) {
-  const { mealType, ingredients } = activity;
-  if (mealType || (ingredients && Object.keys(ingredients).length > 0)) {
+  const { mealType, lineItems } = activity;
+  // Run if: a) Meal type OR b) ALL line items are of type "ingredient" (which should have kilograms as unit)
+  const ingredients = lineItems && lineItems.length && lineItems.filter(l => l.unit === UNIT_KILOGRAMS);
```

Comment:
in principle if the activity is of type meals, and if it has lineitems, then this model should run (I wouldn't chekc for UNIT_KILOGRAMS as we might have portions)

Reply:
removed